### DISCUSSION TITLE
Does `canvas` support background color when draw text? How to set `Paint`

### DISCUSSION THREAD

**Discussion by OliverChao** [rust-skia/rust-skia#898] [Q&A]
I checked the `canvas` specification and it uses `Paint ` to draw.

If there one way to set `Paint` to draw text with the background color?

I am fresh to this project and need some help.

**Comment by pragmatrix** [comment]
Your question is not entirely clear to me. I'll answer based on assumptions:

You can set any color to draw text, but I don't think there is a way to fill the background of the text. This has to be done by measuring the covered area of the text first and then drawing the background (as a number of filled rectangles for example) first before drawing the actual text.
